hyperspy/gui/egerton_quantification.py

```diff
@@ -456,3 +461,20 @@ def apply(self):
         self.interpolated_line = None
         self.reset_span_selector()
         self.find()
+
+
+# For creating a text handler in legend (to label derivative magnitude)
+class AnyObject(object):
```

Comment:
Perhaps... although I do like using overly general names for things that I find online :smirk: 

Should be fixed now